### DISCUSSION TITLE
New to GEM

### DISCUSSION THREAD

**Discussion by Rwillis92** [Mr-MIBonk/M.I.B._More-Incredible-Bash#188] [Q&A]
Trying to figure out how to enable android auto, don't see my firmware. And unfortunately have no realistic way to approach this. current FW MHI2_US_POG11_P3290. I have tried to get the patches downloaded but it refuses to to download as a zip. Thank you for your time.

**Comment by harman-f** [comment]
What does happen, when you try to download?

**Reply by Rwillis92** [reply]
Just shows as a bulk file rather than a zip.

**Reply by harman-f** [reply]
strange, just use the latest POG11 AIO FW update to patch it all https://github.com/harman-f/MHI2_MIB2_AIO_FW_Update_Template
